utils/metrics_encoder_decoder.py

```diff
@@ -137,7 +137,11 @@ def get_metrics_universal(
     """
 
     # Ratings expects a tensor of ratings for multiple users
-    ratings = model_output.unsqueeze(0)
+
+    if len(model_output.shape) < 2:
+        ratings = model_output.unsqueeze(0)
+    else:
+        ratings = model_output
```

Comment:
Maybe this is superflous now? This was needed when batch was 1, but we have changed the output anyway so it might be alright?